cmd_test/constants_test.go

```diff
@@ -0,0 +1,14 @@
+package cmd_test
+
+const (
+	testCollectionName    = "test-collection"
+	qdrantPort            = "6334"
+	qdrantAPIKey          = "00000000"
+	idField               = "__id__"
+	denseVectorName       = "dense_vector"
+	sparseVectorName      = "sparse_vector"
+	batchSize             = 10
+	offsetsCollectionName = "_migration_marker"
+	totalEntries          = 100
+	dimension             = 384
+)
```

Comment:
These are common values shared between the Chroma and Pinecone integration tests.